### PR TITLE
feat: added a stickiness feature for alb aws_alb_target_group

### PR DESCRIPTION
**Pull request by jp08-it**

### CODE REVIEW

File: alb.tf [hunk 1]
```diff
@@ -38,6 +38,18 @@ resource "aws_alb_target_group" "main" {
...
   }
 
+  dynamic "stickiness" {
+    for_each = length(keys(lookup(var.target_groups[count.index], "stickiness", {}))) == 0 ? [] : [
+    lookup(var.target_groups[count.index], "stickiness", {})]
+
+    content {
+      cookie_duration = lookup(stickiness.value, "cookie_duration", null)
+      cookie_name     = lookup(stickiness.value, "cookie_name", null)
+      enabled         = lookup(stickiness.value, "enabled", null)
```
Comment (moritzzimmer) [inline alb.tf:48]: maybe use the default `true` here from https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#stickiness?

File: alb.tf [hunk 2]
```diff
@@ -38,6 +38,18 @@ resource "aws_alb_target_group" "main" {
...
+      type            = lookup(stickiness.value, "type", null)
```
Comment (moritzzimmer) [inline alb.tf:49]: guess we don't know the LB type here to prefil for NLBs like in https://github.com/terraform-aws-modules/terraform-aws-alb/blob/7ed6f047d37510f60c866efc984757fc6b660007/main.tf#L479, do we?

**Review by moritzzimmer — COMMENTED** [review]
thx for the PR @jp08-it, I added two questions to discuss

File: alb.tf (hunk 1 above)
Comment (jp08-it) [reply alb.tf]: i have included the default value to set it to true.

File: alb.tf (hunk 2 above)
Comment (jp08-it) [reply alb.tf:49]: I am not totally sure on this one.

**Review by moritzzimmer — APPROVED** [review]

**Comment by moritzzimmer** [comment]
sorry for the delay @jp08-it, could you please rebase your branch and then we'll merge this to main

**Pull request closed without merging**